tools/terraform_sync_tool/README.md

```diff
@@ -0,0 +1,159 @@
+# Terraform Sync Tool
+
+This directory contains the setup for the Terraform Sync Tool. Terraform Sync Tool was designed to address the schema drifts in BigQuery tables and keep the 
+Terraform schemas up-to-date with the BigQuery table schemas in production environment. Schema drifts occurred when BigQuery Table schemas are updated by newly 
+ingested data while Terraform schema files contain the outdated schemas. Therefore, this tool will detect the schema drifts, trace the origins of the drifts, and alert
+developers/data engineers.
+
+The Terraform Schema Sync Tool fails the build attemps if resource drifts are detected and notifies the latest resource information. Developers and data engineers should be able to update the Terraform resources accordingly. 
```

Comment:
```suggestion
```

Reply:
Updated in https://github.com/GoogleCloudPlatform/bigquery-utils/pull/290/commits/7f469e47bf48003dfc90c9fa406398d1ae927ad8